wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/compose/deserializer/EnvironmentDeserializer.java

```diff
@@ -85,4 +83,21 @@
         */
         throw ctxt.mappingException(format("Unsupported type '%s'.", environment.getClass()));
     }
+
+    /**
+     * Parse "key=value" item.
```

Comment:
It is common practise to separate main description of class from additional description and from params list with `</p>` line. Please consider such formatting.